### PR TITLE
SPUI-1534, SPUI-1537 > Fix recurring buy

### PR DESCRIPTION
**Pull request by Anton-Tesenchuk**

### CODE REVIEW

File: lib/app/shared/features/wallet/view/components/wallet_body/wallet_body.dart [hunk 1]
```diff
@@ -122,7 +122,10 @@ class _WalletBodyState extends State<WalletBody>
                       navigatorPush(
                         context,
                         ShowRecurringInfoAction(
-                          recurringItem: recurringN.recurringBuys[0],
+                          recurringItem: recurringN.recurringBuys.where(
```
Comment (VovaMedvediev) [inline lib/app/shared/features/wallet/view/components/wallet_body/wallet_body.dart:125]: place this logic before the "return" in build function please

File: lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart [hunk 2]
```diff
@@ -29,11 +29,39 @@ class CommonTransactionDetailsBlock extends HookWidget {
     );
 
     return Column(
-      children: [
-        Text(
-          _transactionHeader(transactionListItem, currency),
-          style: sTextH5Style,
+      children: [if (transactionListItem.operationType ==
```
Comment (VovaMedvediev) [inline lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart:32]: start "if" expression from the new line

**Review by VovaMedvediev — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/wallet/view/components/wallet_body/wallet_body.dart (hunk 1 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/wallet/view/components/wallet_body/wallet_body.dart]: done

File: lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart (hunk 2 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart]: done

**Review by VovaMedvediev — APPROVED** [review]

**Pull request merged**